### PR TITLE
Formatted Dynamic Inventory link for Openstack

### PR DESCRIPTION
**Pull request by san7ket**
##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/intro_dynamic_inventory.rst

##### ANSIBLE VERSION

```
2.4devel
```

### CODE REVIEW

**Review by Akasurde — APPROVED** [review]
shipit

File: docs/docsite/rst/intro_dynamic_inventory.rst [hunk 1]
```diff
@@ -247,7 +247,7 @@ Example: OpenStack External Inventory Script
 
 If you use an OpenStack based cloud, instead of manually maintaining your own inventory file, you can use the openstack.py dynamic inventory to pull information about your compute instances directly from OpenStack.
 
-You can download the latest version of the OpenStack inventory script at: https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/openstack.py
+You can download the latest version of the OpenStack inventory script at: `here <https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/openstack.py>`_
```
Comment (dharmabumstead) [inline docs/docsite/rst/intro_dynamic_inventory.rst]: Please remove the "at:"

**Review by dharmabumstead — CHANGES_REQUESTED** [review]

**Comment by san7ket** [comment]
@dharmabumstead done.

**Review by dharmabumstead — APPROVED** [review]
Thanks @san7ket!

**Pull request merged**